spec/controllers/api/drafts_controller_spec.rb

```diff
@@ -79,6 +79,21 @@
     assert_equal(parsed_body['error'], 'unauthorized')
   end
 
+  it 'can not update if the user does not belong to the provider list.' do
```

Comment:
I see a test for preventing users from seeing other providers drafts for update, delete, and query.   But I think we need tests or create and retrieve as well to verify these work as expected.

Reply:
Fixed